src/msg/async/AsyncMessenger.h

```diff
@@ -230,11 +232,19 @@ class AsyncMessenger : public SimplePolicyMessenger {
   bool need_addr = true;
 
   /**
-   * set to bind addresses if bind was called before NetworkStack was ready to
-   * bind
+   * set to bind addresses if bind or bindv were called before NetworkStack
+   * was ready to bind
    */
   entity_addrvec_t pending_bind_addrs;
 
+  /**
+   * set to public addresses (those announced by over the msgr's protocols).
+   * they are stored to handle the cases when either:
+   *   a) bind or bindv were called before NetworkStack was ready to bind,
+   *   b) rebind happens down the road.
```

Comment:
`rebind` is a method, just like `bind` or `bindv` -- I'd stick to the same wording:
```suggestion
   *   b) rebind is called down the road.
```